challenges/Solution 3/Solution3.py

```diff
@@ -0,0 +1,37 @@
+import json
+ids = {}
+i = 1
+rid = 0
+name = ""
+cellphone = ""
+workphone = ""
+email = ""
+address = ""
+b_order = -1
+aorder = -1
+pplan = False
+
+query = ""
+with open("db.txt") as data:
+	d = json.loads(data.read())
+	for r in d:
+		rid = r["Record ID"]
+		name = r["Name"]
+		cellphone = r["Cell Phone"]
+		workphone = r["Work Phone"]
+		email = r["Email"]
+		address = r["Address"]
+		b_order = r["Basic Widget Order"]
+		aorder = r["Advanced Widget Order"]
+		pplan = r["Protection Plan"]
+
+		if email not in ids.keys():
+			ids[email] = i
+			print("INSERT INTO PERSON VALUES("+str(i)+", \""+str(name)+"\", \""+str(email)+"\", \""+str(address)+"\")")
```

Comment:
Given your SQL, how would you plan to manage the relationship between your person table and phone tables? 
Are there any constraints you would enforce on fields in the new model?